report.tex

```diff
@@ -15,6 +24,21 @@
 \bibliographystyle{plain}	% Literaturverzeichnisformat = standard
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%OTHER OPTIONS
+\setlength{\parindent}{0px} %dieser Befehl verhindert das Einrücken eines Absatzes bei einer leeren Codezeile. 
```

Comment:
I would prefer the standard behavior of latex.  Should not be set in template.

Reply:
That's a matter of taste. Maybe a good compromie would be, that it is commented out per default.

Reply:
Good idea!